gateway-js/src/index.ts

```diff
@@ -559,9 +556,7 @@ export class ApolloGateway implements GatewayInterface {
   ): void {
     this.queryPlanStore.clear();
     this.apiSchema = coreSchema.toAPISchema();
-    this.schema = addExtensions(
-      wrapSchemaWithAliasResolver(this.apiSchema.toGraphQLJSSchema()),
```

Comment:
Note to reviewers: `wrapSchemaWithAliasResolver` was needed due to the use of GraphQL-js execution. Namely, when GraphQL-js execute is passed an object, it expects that object to match the schema and so to use field names from the schema. But the internal data we gather during query plan execution will have data under the "response name", and so under aliases when aliases was used. That method made sure things were still working.

The new code is written to use response name directly, so no need for that.